src/sentry/digests/utilities.py

```diff
@@ -47,23 +54,16 @@ def get_personalized_digests(target_type, project_id, digest, user_ids):
             yield user_id, digest
 
 
-def get_event_from_groups_in_digest(digest):
-    """
-    get_event_from_groups_in_digest(digest: Digest)
-
-    Gets the first event from each group in the digest
-    """
-    events = []
+def get_event_from_groups_in_digest(digest: Digest) -> Iterable[Event]:
```

Comment:
Shouldn't this be `Set[Event]`?

Reply:
I prefer to use more general types like `Sequence` and `Iterable` so that we aren't locked into an implementation inside the function. We just end up iterating over the events in `get_personalized_digests()` so I made it and `Iterable`.

Reply:
I don't fully understand your comment. If that were the case wouldn't you also be using `Iterable` almost everywhere and not `Sequence` at all? However we are using `Sequence`.

We also check `item in set` later not just looping through all events. Personally I think it is useful to mark as set so that we remember that we are dealing with unordered data etc, but this may just be my opinion.